### PR TITLE
Add release instructions to README

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -503,3 +504,18 @@ total_failures = wavefront_sender.get_failure_count()
...
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file. There you will find the version number for example `1.8.10`.
+4. From the master branch, run `git tag -a -m 'v1.8.10' 'v1.8.10'` replacing v1.8.10 with the new version found in step 3. Remember to keep the `v` prefix. So if the version in step 3 is `1.8.12` you would use `v1.8.12` in the git tag command in this step.
+5. Push the tag up to Github `git push upstream v1.8.10`.
```
Comment (ustinov) [inline README.md:514]: The new tag can actually be created during drafting the release

File: README.md [hunk 2]
```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
```
Comment (shavidissa) [inline README.md:510]: Merge all the changes that need to go into the release to the master branch.

File: README.md [hunk 3]
```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
...
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file. There you will find the version number for example `1.8.10`.
+4. Log into Github, click on **Releases** in the about section on the right, and click on **Draft a new release.**
```
Comment (shavidissa) [inline README.md:513]: Log into Github, click  **Releases** on the right, and click **Draft a new release.**

File: README.md [hunk 4]
```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file. There you will find the version number for example `1.8.10`.
```
Comment (shavidissa) [inline README.md:512]: Search for `version=` in the file to find the version number. For example, `1.8.10`.

File: README.md [hunk 5]
```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
...
+4. Log into Github, click on **Releases** in the about section on the right, and click on **Draft a new release.**
+5. For **version**, choose the version you found in step 3, but prefix it with `v` for example `v1.8.10
```
Comment (shavidissa) [inline README.md:514]: For **version**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.101`
 
 The code needs to be closed :) 
 
 Can we add a screenshot here? I tried following the steps but couldn't do it coz am not familiar with the process. A screenshot will help :)

File: README.md [hunk 6]
```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
...
+6. Provide a short but descriptive title for the release.
+7. Fill in the details of the release. Please copy the markdown from the previous release and follow the same format.
+8. Click on **Publish release.**. This will automatically start an action that will publish the the release to pypi.org.
```
Comment (shavidissa) [inline README.md:517]: Click **Publish release.** to start publishing the release to pypi.org.

File: README.md [hunk 7]
```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
...
+9. From the Github top navigation bar of this project, click on the **Actions** tab. On the first line of the list of workflows, you should see a workflow running which will publish your release to pypi.org
```
Comment (shavidissa) [inline README.md:518]: From the Github top navigation bar of this project, click the **Actions** tab. On the first line of the list of workflows, you should see a workflow running that will publish your release to pypi.org

File: README.md [hunk 8]
```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
...
+10. When the workflow from step 9 has a green checkmark by it, go to [pypi.org](https://pypi.org/project/wavefront-sdk-python/) to verify that the latest version has been released.
```
Comment (shavidissa) [inline README.md:519]: When the workflow from step 9 has a green checkmark next to it, go to [pypi.org](https://pypi.org/project/wavefront-sdk-python/) and verify that the latest version is published.

**Review by shavidissa — CHANGES_REQUESTED** [review]
Hi,

I have some minor suggetions! Thank you for involving me :) Great work :)

File: README.md (hunk 2 above)
Comment (keep94) [reply README.md:510]: Done.

File: README.md (hunk 3 above)
Comment (keep94) [reply README.md:513]: Done.

File: README.md (hunk 4 above)
Comment (keep94) [reply README.md:512]: Done.

File: README.md (hunk 7 above)
Comment (keep94) [reply README.md:518]: Done.

File: README.md (hunk 8 above)
Comment (keep94) [reply README.md:519]: Done.

File: README.md (hunk 6 above)
Comment (keep94) [reply README.md:517]: Done.

File: README.md [hunk 9]
```diff
@@ -503,3 +504,19 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Merge all the changes that need to go into the release to the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file to find the version number for example `1.8.10`.
+4. Log into Github, click **Releases** on the right, and click **Draft a new release.**
```
Comment (shavidissa) [inline README.md:513]: Log into Github, click **Releases** on the right, and click **Draft a new release**.

File: README.md [hunk 10]
```diff
@@ -503,3 +504,19 @@ total_failures = wavefront_sender.get_failure_count()
...
+5. For **Choose a tag**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.10`. You will need to enter the version where it says **Find or create new tag.**
+
+<img src="images/choose-version.png" alt="A diagram that shows how to choose version"/>
+
+6. Provide a short but descriptive title for the release.
+7. Fill in the details of the release. Please copy the markdown from the previous release and follow the same format.
+8. Click **Publish release.** to start publishing the release to pypi.org.
+9. From the Github top navigation bar of this project, click the **Actions** tab. On the first line of the list of workflows, you should see a workflow running that will publish your release to pypi.org
```
Comment (shavidissa) [inline README.md:521]: Fullstop at the end of the sentence. :)

File: README.md [hunk 11]
```diff
@@ -503,3 +504,19 @@ total_failures = wavefront_sender.get_failure_count()
...
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Merge all the changes that need to go into the release to the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file to find the version number for example `1.8.10`.
+4. Log into Github, click **Releases** on the right, and click **Draft a new release.**
+5. For **Choose a tag**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.10`. You will need to enter the version where it says **Find or create new tag.**
```
Comment (shavidissa) [inline README.md:514]: You need to enter the version where it says **Find or create new tag**.

**Review by shavidissa — APPROVED** [review]
Minor changes requested :) 
LGTM

File: README.md (hunk 9 above)
Comment (keep94) [reply README.md:513]: I am sorry, but it seems that your suggestion matches what I typed in.  Not sure what I need to change.

Comment (keep94) [reply README.md:513]: Never mind, I see now that the period at the end of the sentence should not be in bold.

File: README.md (hunk 10 above)
Comment (keep94) [reply README.md:521]: Done.

File: README.md (hunk 11 above)
Comment (keep94) [reply README.md:514]: Done.

**Review by ustinov — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (keep94) [reply README.md]: I fixed the directions.

**Pull request merged**